src/rpg_database/character_sheet/Background.java

```diff
@@ -20,6 +20,7 @@
 	private final String text;
 	private final HashSet<BaseClasses> baseClasses;
 	private final LanguagesSet languages;
+	private final Race race;
```

Comment:
ennek valamiféle collectionnek kellene lennie, lsd lejjebb. Lehet az allowedRaces név az célszerűbb is lenne emiatt

Reply:
ok.